frontend/public/components/masthead-toolbar.jsx

```diff
@@ -213,23 +282,57 @@ class MastheadToolbar_ extends React.Component {
         externalLink: true,
       };
     });
+
+    return {
+      section: '',
```

Comment:
It's unclear why we can't return an array from this function and just add the sections in `reduce` above?

Same comment above where help menu actions are given a `section: ''`. It's a a little confusing because it looks like they can have a section when they can't.

Reply:
Oh, is it because we're using the application launcher for the other menus now, too?

Reply:
correct. And in Mobile mode this gives the separators between the sections.